### ISSUE TITLE
[Feature request] MistraL-7B support

### ISSUE THREAD

**Issue by pipilurj** [haotian-liu/LLaVA#763]
### feature

Hi! Mistral-7B seems to be a great opensource LLM, do you have plans on integrating it into LLAVA? Thanks!

**Comment by haotian-liu** [comment]
Yes, we are working on integrating Mistral-7B to the code base, and since it requires a later transformers version, we are working to make sure that the existing code does not break when we upgrade, as there is a major change in terms of tokenization after we upgrade.

**Comment by liyang-7** [comment]
There seems to be a problem with directly using the current tokenization to train Mistral-7B. 
Can you elaborate on what aspects need to be modified for Mistral-7B or when the code for training Mistral-7B can be open-sourced?
Thanks!

**Comment by liyang-7** [comment]
Can we exchange WeChat for further convenience communication in Mistral or Zephyr training?  ID：tiancaili7

**Comment by BlueBlueFF** [comment]
any update？

**Comment by ChunyuanLI** [comment]
https://huggingface.co/SkunkworksAI/BakLLaVA-1
